lib/posix/posix.nim

```diff
@@ -686,6 +772,13 @@ proc getrusage*(who: cint, rusage: ptr Rusage): cint
 proc bsd_signal*(a1: cint, a2: proc (x: pointer) {.noconv.}) {.
   importc, header: "<signal.h>".}
 proc kill*(a1: Pid, a2: cint): cint {.importc, header: "<signal.h>".}
+
+proc kill*(pid: Pid, signal: int) =
+  ## Sends a signal to a running process.
+  ## Raise exception in case of failure e.g. process not running.
+  if kill(pid, signal.cint) != 0:
+    raise newException(OSError, $strerror(errno))
+
```

Comment:
This overload is a breaking change. Programs that have been written to test the return value won't compile anymore.